src/MSBuild/Microsoft.Build.CommonTypes.xsd

```diff
@@ -1796,7 +1796,7 @@ elementFormDefault="qualified">
     </xs:element>
     <xs:element name="VersionPrefix" type="msb:StringPropertyType" substitutionGroup="msb:Property" >
       <xs:annotation>
-        <xs:documentation><!-- _locID_text="VersionPrefix" _locComment="" -->Prefix that can indicate additional information at the end of the verion (i.e. beta)</xs:documentation>
+        <xs:documentation><!-- _locID_text="VersionPrefix" _locComment="" -->Prefix that can indicate additional information at the start of the version (i.e. beta)</xs:documentation>
```

Comment:
Agreed this is not a good description for this property.

This is the best documentation I can find that explains these properties:  https://github.com/dotnet/cli/issues/5573#issuecomment-277377729